lib/src/clojure_lsp/kondo.clj

```diff
@@ -16,7 +16,7 @@
 (defn clj-kondo-version []
   (string/trim (slurp (io/resource "CLJ_KONDO_VERSION"))))
 
-(def clj-kondo-analysis-batch-size 50)
+(def clj-kondo-analysis-batch-size 150)
```

Comment:
Something I noticed with this change, is that we have a little bit less feedback on the progress bar of editor and CLI when analyzing external deps since we have fewer batches, this is ok since performance is way more important than feedback.
I wonder if we could use some kondo hook to improve that, for example: we know there are 80 paths on classpath passed to kondo, every path linted we update the progress from a kondo hook or something, sounds like something nice to explore later

Reply:
and we could use this for the 95% (Project code analysis), making it more real with more precise updates

Reply:
That's true... I like the idea of a progress hook. clj-kondo already has a mechanism for printing dots "." when it's in its `:text` output format, so at least a little of the groundwork is there.

Reply:
started a discussion [here](https://clojurians.slack.com/archives/CHY97NXE2/p1651283825804139)